easybuild/easyconfigs/w/wxPython/wxPython-4.2.0-foss-2021b.eb

```diff
@@ -0,0 +1,77 @@
+# This easyconfig was created by the BEAR Software team at the University of Birmingham.
+# Updated to version 4.2.0
+# J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'PythonBundle'
+
+name = 'wxPython'
+version = '4.2.0'
+
+homepage = "https://www.wxpython.org/"
+description = """Wraps the wxWidgets C++ toolkit and provides access to the user interface portions of the wxWidgets 
+API, enabling Python applications to have a native GUI on Windows, Macs or Unix systems, with a native look and feel 
+and requiring very little (if any) platform specific code."""
+
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+# sources = ['%(name)s-%(version)s.tar.gz']
+# patches = [
+#    'wxPython-4.1.1_fix_install_path.patch',
+#    'wxPython-4.1.1_use_bang_env_python.patch',
+# ]
+# checksums = [
+#    '663cebc4509d7e5d113518865fe274f77f95434c5d57bc386ed58d65ceed86c7',  # wxPython-4.2.0.tar.gz
+#    'f13743877deddbf525bbb3f81c8f7a6b0c2dbf1333595926f653f696999e31ce',  # wxPython-4.1.1_fix_install_path.patch
+#    'c355c60a8cce3018fc0c30ffc78623efd2481e9baf33673e22a57863e1a3ac87',  # wxPython-4.1.1_use_bang_env_python.patch
+# ]
+
+dependencies = [
+    ('Python', '3.9.6'),
+    ('SciPy-bundle', '2021.10'),
+    ('Pillow', '8.3.2'),
+    ('libpng', '1.6.37'),
+    ('zlib', '1.2.11'),
+    ('libjpeg-turbo', '2.0.6'),
+    ('LibTIFF', '4.3.0'),
+    ('expat', '2.4.1'),
+    ('GTK3', '3.24.31'),
+    ('GLib', '2.69.1'),
+    ('GConf', '3.2.6'),
+    ('GST-plugins-base', '1.18.5'),
+    ('pkg-config', '0.29.2'),
+    ('Mesa', '21.1.7'),
+    ('libGLU', '9.0.2'),
+    ('LibSoup', '3.0.7'),
+    ('WebKitGTK+', '2.37.1'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('attrdict', '2.0.1', {
+        'checksums': ['35c90698b55c683946091177177a9e9c0713a0860f0e049febd72649ccd77b70'],
+    }),
+    (name, version, {
+        'source_tmpl': '%(name)s-%(version)s.tar.gz',
+        'patches': ['wxPython-4.1.1_fix_install_path.patch'],
+        'checksums': [
+            '663cebc4509d7e5d113518865fe274f77f95434c5d57bc386ed58d65ceed86c7',  # wxPython-4.2.0.tar.gz
+            'f13743877deddbf525bbb3f81c8f7a6b0c2dbf1333595926f653f696999e31ce',  # wxPython-4.1.1_fix_install_path.patch
+        ],
+        'sanity_check_paths': [{
+            'files': ['bin/wxget', 'lib/python3.9/site-packages/wx/libwx_baseu-3.2.%s' % SHLIB_EXT,
+                      'lib/python3.9/site-packages/wx/libwx_gtk3u_xrc-3.2.%s' % SHLIB_EXT,
+                      'lib/python3.9/site-packages/wx/libwx_baseu_xml-3.2.%s' % SHLIB_EXT,
+                      'lib/python3.9/site-packages/wx/libwx_gtk3u_richtext-3.2.%s' % SHLIB_EXT],
+            'dirs': []
+        }]
```

Comment:
```suggestion
        'sanity_check_paths': {
            'files': ['bin/wxget', 'lib/python3.9/site-packages/wx/libwx_baseu-3.2.%s' % SHLIB_EXT,
                      'lib/python3.9/site-packages/wx/libwx_gtk3u_xrc-3.2.%s' % SHLIB_EXT,
                      'lib/python3.9/site-packages/wx/libwx_baseu_xml-3.2.%s' % SHLIB_EXT,
                      'lib/python3.9/site-packages/wx/libwx_gtk3u_richtext-3.2.%s' % SHLIB_EXT],
            'dirs': []
        }
```

Reply:
One day, I promise, I will learn that! Thanks